### DISCUSSION TITLE
Dynamic Dialog with reactive Angular Form reloads the whole page when DynamicDialogRef#close() is called

### DISCUSSION THREAD

**Discussion by ThoSap** [primefaces/community#1478] [PrimeNG]
When you have an Angular reactive form inside a component, that is opened with a Dynamic Dialog and closed with a custom close or about button using `DynamicDialogRef#close()`, the whole page gets refreshed.
Wrapping the `DynamicDialogRef#close()` in a `setTimeout(() =>  this.dynamicDialogRef.close(profile)), 0)` is currently somehow a workaround so the page is not reloaded.

See GitHub issue https://github.com/primefaces/primeng/issues/14685

In the StackBlitz of the original GH Issue and linked below it is somehow needed to click close multiple times, but for me the page reload is triggered if I click the close or abort button only once.
https://stackblitz.com/edit/stackblitz-starters-zhaboa?file=src%2Fapp-dialog%2Fview%2Fview.component.ts

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for raising the issue we'll review the issue in v17.13.0 or higher releases.
